### ISSUE TITLE
Error/Traceback when quitting gping with CTRL+C

### ISSUE THREAD

**Issue by KoesterD** [orf/gping#10]
I use Windows 8.1 with Python 3.5.0 and get the following error when CTRL+Cing gping:
```
Traceback (most recent call last):
  File "c:\users\dennis_lpt\appdata\local\programs\python\python35\lib\runpy.py", line 170, in _run_module_as_main
    "__main__", mod_spec)
  File "c:\users\dennis_lpt\appdata\local\programs\python\python35\lib\runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "C:\Users\dennis_lpt\AppData\Local\Programs\Python\Python35\Scripts\gping.exe\__main__.py", line 9, in <module>
  File "c:\users\dennis_lpt\appdata\local\programs\python\python35\lib\site-packages\gping\pinger.py", line 251, in run
    for ping in it(url):
  File "c:\users\dennis_lpt\appdata\local\programs\python\python35\lib\site-packages\gping\pinger.py", line 213, in _windows
    line = ping.stdout.readline().decode()
KeyboardInterrupt
```

It's just an esthetic issue, not impairing anything at all. Thought it might be worth reporting.

**Comment by Robert-McGinley** [comment]
Probably just need to catch KeyboardInterrupt and run sys.exit(). Should be a quick fix.

**Comment by orf** [comment]
I fixed this in 28884a9352903170cc19510cb2e468a8acec1a32, thanks for reporting :)
